index.ts

```diff
@@ -79,12 +79,13 @@ export function missingTranslationHandler() {
 })
 export class NgJhipsterModule {
     static forRoot(providedConfig: ModuleConfig): ModuleWithProviders {
-        ConfigHelper.setModuleConfigOptions(providedConfig);
```

Comment:
Make  sure To test it in a real project like angular2-starter.  I think i implemented it this way because it wouldn't work with just an export but I'm not sure.